### PR TITLE
Ns rse/75 remove git install

### LINKED ISSUE #75: Remove installation of Git from setup instructions
GitKraken comes with Git baked in and so the need to install standalone Git is now superflous.

Pointing people via links on how to setup and use other IDEs or install Git on its own could be moved to FAQs/Links section.

### PR DESCRIPTION
**Pull request by ns-rse**
Closes #75

As discussed because GitKraken [does not require any Git tools](https://help.gitkraken.com/gitkraken-client/how-to-install/) participants only need to install GitKraken and they should be good to go.

**However** if someone could confirm this (particularly under Windows) that would be great. 

Under Arch Linux I've checked the GitKraken Terminal with `which git` and its picking up the system one (`/usr/bin/bit`) which raises a slight cause for concern.

### CODE REVIEW

**Comment by davidwilby** [comment]
>Under Arch Linux I've checked the GitKraken Terminal with which git and its picking up the system one (/usr/bin/bit) which raises a slight cause for concern.

Same under Ubuntu linux.

File: content/setup/optional/_index.en.md [hunk 1]
```diff
@@ -3,14 +3,117 @@ title: Optional
...
+<li>Quit command prompt by typing <code>exit</code> then pressing [Enter]</li>
+</ol>
+</li>
+</ol>
+<p>This will provide you with both Git and Bash in the Git Bash program.</p>
+
+
+
+
+#### Mac OS X
```
Comment (bobturneruk) [inline content/setup/optional/_index.en.md:65]: Likewise, could this be linked out to?

**Comment by EdwinB12** [comment]
Can confirm on Windows, GitKraken comes with git.

**Comment by davidwilby** [comment]
> Can confirm on Windows, GitKraken comes with git.

Out of interest, how have you confirmed this? Same as above?


Is this kind of moot if on Windows GitKraken comes with git? Do all linux distros have git installed? Ubuntu does, I think.

According to this thread, MacOS doesn't though https://developer.apple.com/forums/thread/672087

**Comment by EdwinB12** [comment]
The laptop I tested it on didn't have git installed. After installing gitkraken, I could use all of gitkraken git capabilities like cloning etc.

**Comment by ns-rse** [comment]
@davidwilby I'd hazard a guess that almost all Linux distros come with Git (but if you're using something like Arch or Gentoo which might not then you would likely have installed it anyway or know how to).

@EdwinB12 Thanks for the confirmation on Windows (and OSX @davidwilby ). I think as GitKraken bundles it (and states as such) we can safely move instructions for installing Git to the **Optional** as this PR has done.

That just leaves the question of whether we want to include the existing instructions there or as @bobturneruk suggests removing them completely and linking out to them. I don't mind them staying in myself but happy to change to links to installation instructions if others think so. @annakrystalli do you have any preference?

File: content/setup/optional/_index.en.md [hunk 2]
```diff
@@ -3,14 +3,117 @@ title: Optional
 weight: 5
 ---
 
-During the collaborative exercise, we will collate materials from each participant which will feed into an R analysis. If you would like to run this analysis yourself locally, you will need to have **R** and **Rstudio** installed. This however is optional and not a requirement for you to complete the workshop.
+### Git
+
+If you wish to install Git and use it outside of GitKraken follow the installation instructions according to your system:
+
+#### Windows
```
[the remark this answers is not in the document]
Comment (EdwinB12) [reply content/setup/optional/_index.en.md:10]: I would agree with Bob. I would replace this entire selection by pointing people to this link: 

https://git-scm.com/book/en/v2/Getting-Started-Installing-Git

File: content/setup/optional/_index.en.md (hunk 1 above)
Comment (EdwinB12) [reply content/setup/optional/_index.en.md:65]: As above: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git

**Pull request merged**